### ISSUE TITLE
Make subliminal run on libreelec

### ISSUE THREAD

**Issue by jdeluyck** [queeup-archives/repository.queeup#5]
Would you consider adding

sys.path.append('/storage/.kodi/addons/shell.python.subliminal/resources/lib')

in subliminal, so that it would also run without issues on LibreELEC?

**Comment by queeup** [comment]
You mean change directory .xmbc to .kodi? It works for me on LibreELEC because I have symbolic link .kodi to .xbmc. But I will take care of it. Thx to remind me that. Sorry for late answer...

**Comment by jdeluyck** [comment]
Basically. I put a symlink for now - but it's a "cleaner" solution.

**Comment by queeup** [comment]
Next version of subliminal I will fix that. Thx again. Until it fixed I will leave it open this issue.

**Comment by msdos** [comment]
> You mean change directory .xmbc to .kodi?

You don't need to change it, you can keep both.

```
sys.path.append('/storage/.xbmc/addons/shell.python.subliminal/resources/lib')
sys.path.append('/storage/.kodi/addons/shell.python.subliminal/resources/lib')
```

**Comment by queeup** [comment]
Fixed with version 2.1.0 of subliminal: 0bd32cd
